scripts/execute-deepnote.py

```
@@ -0,0 +1,891 @@
+            elif package == 'scikit-learn':
+                f.write("scikit-learn>=1.3.0\n")
+            elif package == 'matplotlib':
+                f.write("matplotlib>=3.7.0\n")
+            elif package == 'seaborn':
+                f.write("seaborn>=0.12.0\n")
+            else:
+                f.write(f"{package}\n")
+    
+    return True
+
+def execute_deepnote_file(filepath: str, kernel_id: str = None):
```

Ruff (0.14.3)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)